examples/README.md

```diff
@@ -0,0 +1,125 @@
+
+1. Navigate to the Google Drive folder where the models are located: https://drive.google.com/drive/u/2/folders/1Av3TyvfiOIy5PbTMgsZiB6MRKNmzdWCw
+2. If you cannot access the files, email [Kevin Bonhan](mailto:kbonham@wellesley.edu) or [Guilherme Bottino](mailto:gz101@wellesley.edu) regarding granting you access. You may also use the "request access" functionality on Google Drive.
+3. Download the models and extract them to the `models/` folder inside the package root directory.
+
+## Running the example scripts
+
+### Training your own models
+
+The first example script is `examples/train_univariate_randomforest.jl`. This script takes five **mandatory** arguments:
+
+`julia examples/train_univariate_randomforest.jl {-c | -r} input_csv input_cols output_col output_filename`
+
+1. The first argument tells the script to run in either Regression or Classification mode.
+
+2. The second argument is a community profile (with subjects/samples as rows, and features/predictors as columns) on the CSV format. The script expects a header for the CSV amd may fail if a header line is not provided. Please refer to the provided `examples/example_taxonomic_profile.csv` for formatting.
+
+3. The third argument tells the script which columns contain the actual model inputs. This can be provided as a range (ex: `1:100`, `6:550`) or a column vector (ex: `[1,2,3,6,7,8,9,10,12]`). This is useful in case your model contains metadata columns that are irrelevant for prediction
+   
+4. The fourth argument tells the script which column contains the expected output for supervised training. Since this is a univariate predictor, this argument takes a single integer number.
+
+5. The fifth argument is the path to the file that will store the trained model. We elected the `JLD2` Julia package to handle the distribution of models, hence the script expects the `.jld` file extension.
+
+From the package root directory, you can use the example data provided to run the script. Try the commands
+
+```
+julia examples/train_univariate_randomforest.jl -c examples/example_taxonomic_profile.csv 6:554 4 examples/example_classifier_output.jld
+julia examples/train_univariate_randomforest.jl -r examples/example_taxonomic_profile.csv 6:554 4 examples/example_regressor_output.jld
+```
+
+To train either a classifier or a regressor.
+
+#### hyperparameter tuning
+
+To keep execution times low, we ship the example script with a sparse grid for hyperparameter optimization. As a consequence, the output models might have low predictive performance. If the test run works, you may want to increase the density of the tuning grid, while also adjusting its boundaries. In order to do so, edit the following lines on the training script:
```

Comment:
For a "production" version of this, we can't do this. Again, not worth changing now, but we'll need to put these into a config toml file or something

Reply:
Please open an issue so we can keep track of this

Reply:
Done.